src/uwtools/config/formats/yaml.py

```diff
@@ -145,3 +141,52 @@ def get_format() -> str:
         Returns the config's format name.
         """
         return FORMAT.yaml
+
+    # Private methods
```

Comment:
I relocated some YAML-related code from `uwtools.config.support` to the `YAMLConfig` class. I half expected a circular-dependency issue to arise in the refactor, and assumed that that's the reason the code lived where it did, but was pleasantly surprised to find no issues incorporating it into the class where it seems like it belongs.